galaxy_ng/app/api/ui/viewsets/user.py

```diff
@@ -11,6 +11,14 @@
 
 
 class UserFilter(filterset.FilterSet):
+    username = filters.CharFilter(field_name='username', lookup_expr='icontains')
```

Comment:
We can't accept these changes because they are technically a breaking change (by changing the behaviour of an existing filter) and they aren't consistent with the rest of our filtering on the API.

If you want to update the UI to use icontains filtering instead of exact filtering, then the UI should be updated to perform `?<field_name>__icontains=foo` lookups instead of changing the default behavior of `?<fieldname>=foo` lookups.

There are cases where clients may need to look up a result using an exact match and replacing the exact filter with one that performs an icontains lookup under the hood will break that.

Reply:
@newswangerd so https://issues.redhat.com/browse/AAH-355 is an invalid issue?

Reply:
@rochacbruno it's mostly valid. The changes should happen on the UI tho, not the API. the API already supported icontains lookups, you just had to do `?username__icontains` instead of `?username` for example.

Reply:
ok, so I think we can close this PR ? @ShaiahWren

Reply:
@rochacbruno close the pr and DO NOT MERGE. haha. My new pr for this issue is over on https://github.com/ansible/ansible-hub-ui/pull/670